Automattic/wp-calypso
# Themes Tiers: Live preview aware of Personal Themes

Related to:
- https://github.com/Automattic/dotcom-forge/issues/4597

## Proposed Changes
The scope of this PR is just to enable the Personal Tier for Theme Live Previewing, the texts and checkout URLs are out of scope and should not be considered.
In this PR, when you are on a Free Plan, Personal themes are now Live Previewable instead of being able to activate. Once you open the Live Preview you will see all the upsells and banners (they still reference Premium, the texts will be addressed on a separate PR)
* The Live Previewer is now aware of Personal Themes
* Once we release Theme Tiers we will need to refactor the Live Preview since it can be greatly simplified by using the theme_tier.feature prop instead of resolving the feature based on the type and then checking the active features.

## Testing Instructions


* Sandbox `widgets.wp.com`
* Open the Calyspo live link.
* Go to the LiTS and add `?flags=themes/tiers`.
* Filter by Personal Themes and click on the details of any Personal Theme.
* On the Theme Details page, click on the Live Preview button.
* You should see the same elements, upsells, and CTAs as if you were previewing a Personal theme. (Remember, the texts are not part of this PR for simplicity reasons)

![Captura de pantalla 2023-12-21 a las 13 54 55](https://github.com/Automattic/wp-calypso/assets/1989914/27edd80a-4773-44d9-90a5-41c9e966befe)
![Captura de pantalla 2023-12-21 a las 13 55 24](https://github.com/Automattic/wp-calypso/assets/1989914/0eace8bc-52e9-455f-bd21-312d2d54203a)
![Captura de pantalla 2023-12-21 a las 13 55 32](https://github.com/Automattic/wp-calypso/assets/1989914/efdaabc3-ee33-4f61-b0d7-9d1a0c5326b0)

## Pre-merge Checklist
